### DISCUSSION TITLE
Waiting to post examples that otherwise need a legacy fallback explanation.

### DISCUSSION THREAD

**Discussion by estelle** [mdn/mdn-community#491] [Content]
#29703 is a case that demonstrates that, while we do want to make sure we document everything as soon as possible, we may want to wait a few weeks before updating examples to highlight the newest released features if they need to have a legacy support exception or temporary polyfill. 

In a few releases, or possibly a few weeks, we'll be able to revert #29703, but that won't be on anyone's radar anymore, and the "legacy support" statements will be around for a couple of years. 

My thought is that if we don't need to provide a fallback, especially if there is "experimental" listed next to the feature, we can post new features immediately, but if we know it will be supported everywhere almost immediately, instead of adding "for legacy support" statements, if we could wait a few weeks to update the examples  (not the content explaining it, but the general examples that use the feature as if fully supported) that would be good. Browser tech writers are incentivized to write about their latest features, so they will likely be willing to write their content still, even if we push out publishing a few weeks. They may actually be happy not to have to write the legacy stuff. But no one is incentivized to remove the legacy language when it is no longer needed.

**Comment by chrisdavidmills** [comment]
I agree with your sentiment here. To me, it depends on a couple of things:

1. How long the need for legacy is likely to be around. In the case you cite, yup, it would only be a few weeks until Chrome 119 is out, so I possibly could've waited. However, it isn't just the case that as soon as a new release version hits, everyone immediately updates. It takes longer than you might think, for various reasons, especially on mobile platforms. So need for legacy might be longer than first thought.
2. How complex the legacy is. In the cited case, it is very simple, and developers can easily identify what bits they can include or remove from the code example in future when the legacy is perhaps not needed? However, for more complex cases we need to think more carefully.
